bindata/network/multus/multus.yaml

```diff
@@ -0,0 +1,78 @@
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: multus
+  namespace: multus
+  annotations:
+    kubernetes.io/description: |
+      This daemon set launches the Multus networking component on each node.
+spec:
+  selector:
+    matchLabels:
+      app: multus
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: multus
+        component: network
+        type: infra
+        openshift.io/component: network
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
+      hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/os: linux
+      tolerations:
+      - operator: Exists
+      serviceAccountName: multus
+      initContainers:
+      - name: cni-plugins-supported
+        image: {{.CNIPluginsSupportedImage}}
+        command: ["/bin/sh"]
+        args: ["-c", "cp -rf /usr/src/plugins/bin/* /host/opt/cni/bin"]
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /host/opt/cni/bin
+          name: cnibin
+      - name: cni-plugins-unsupported
+        image: {{.CNIPluginsUnsupportedImage}}
+        command: ["/bin/sh"]
+        args: ["-c", "cp -rf /usr/src/plugins/bin/* /host/opt/cni/bin"]
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /host/opt/cni/bin
+          name: cnibin
+      containers:
+      - name: kube-multus
+        image: {{.MultusImage}}
+        command: ["/entrypoint.sh"]
+        args:
+        - "--multus-conf-file=auto"
+        - "--multus-kubeconfig-file-host=/etc/kubernetes/cni/net.d/multus.d/multus.kubeconfig"
+        - "--namespace-isolation=true"
+        resources:
+          requests:
+            cpu: "100m"
+            memory: "50Mi"
+          limits:
+            cpu: "100m"
+            memory: "50Mi"
```

Comment:
Drop the `limits` section unless (a) there's a chance that the pod will attempt to use a lot of memory/CPU in some circumstances, and (b) you actually want to thwart it when that happens. (You probably just copied this from the openshift-sdn daemonset, and the `limits` have been removed there since then.)